### PR TITLE
FIX: Using excitation type "winding" instead of "winding group"

### PR DESCRIPTION
**Pull request by IreneWoyna**

### CODE REVIEW

**Comment by IreneWoyna** [comment]
@gmalinve Maxwell throws an error and doesn't create a circuit already when I'm calling the method: 
wdg_groups = self.excitations_by_type["Winding Group"]
(also when I'm actually not using wdg_groups)

That's why I commented it in the code.

File: pyaedt/maxwell.py [hunk 1]
```diff
@@ -1982,7 +1982,8 @@ def create_external_circuit(self, circuit_design=None):
 
         circuit = MaxwellCircuit(design=circuit_design)
 
-        wdgs = self.excitations_by_type["Winding Group"]
+        wdgs = self.excitations_by_type["Winding"]
```
Comment (gmalinve) [inline pyaedt/maxwell.py:1985]: @IreneWoyna you should check both cases :)

Comment (IreneWoyna) [reply pyaedt/maxwell.py:1985]: see my comment below, Maxwell throws an error already when I try to check the "winding group" and doesn't create the circuit...

Comment (gmalinve) [reply pyaedt/maxwell.py:1985]: @IreneWoyna yes but you should check that "Winding" or "Winding Group" are keys of the excitations_by_type dictionary. If yes you get you excitations as: self.excitation_by_type[excitation_type].

Comment (IreneWoyna) [reply pyaedt/maxwell.py]: @gmalinve now it should work

**Review by Samuelopez-ansys — DISMISSED** [review]

**Review by gmalinve — APPROVED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**